app/screens/login/login.js

```diff
@@ -93,6 +95,8 @@ class Login extends PureComponent {
             });
         }
 
+        this.props.actions.autoUpdateTimezone(getDeviceTimezone());
```

Comment:
would this update the timezone every time you login?

do we need to do the same when logging in with sso?